core/data/inline.rng

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+
+<grammar xmlns="http://relaxng.org/ns/structure/1.0"
+         datatypeLibrary="http://www.w3.org/2001/XMLSchema-datatypes">
+
+    <define name="libero.text.limited">
```

Comment:
Not sure about the name `limited`. Meant to distinguish from `full`, where you could have any inline element (here you probably don't want things like a link).

Reply:
Is the subset of elements `graphical` or related to `formatting`?

Reply:
Need to definitely look at grouping inline elements, like https://jats.nlm.nih.gov/archiving/tag-library/1.1/pe/emphasis.class.html and http://www.tei-c.org/release/doc/tei-p5-doc/en/html/ref-model.pPart.editorial.html.

Reply:
`limited` is left over from the rather naive form in the skeleton (https://github.com/libero/walking-skeleton/blob/master/schemas/data/core/inline.rng#L86-L120).